atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
I actually plan to have a look at this. Leave it to me then, I'll make it work across all the different scenarios.